### DISCUSSION TITLE
Build failing for Kafka connect

### DISCUSSION THREAD

**Discussion by raju-lv** [strimzi/strimzi-kafka-operator#5569] [Q&A]
Hi, I am deploying strimzi kafka-connect with 2.7.0 and operator version 0.23. While deploying kafka connect with 3 replicas build failing due to following issue. I have tried for both public and private repositories, for both build is failing. Any suggestions in configurations will be helpful. Providing configurations of kafka-connect in below file. Thanks.

[kafka-connect-config.txt](https://github.com/strimzi/drain-cleaner/files/7171565/kafka-connect-config.txt)

------------------------------------
kafka-connect-build pod logs:
-->kubectl logs my-connect-cluster-connect-build -n kafka-ethonag
error checking push permissions -- make sure you entered the correct tag name, and that you are authenticated correctly, and try again: checking push permission for "<registry name>/strimzi-kafka-connect-ng-post:2.7.0": creating push check transport for <registry name>: Get "<registry name>/v2/": x509: certificate signed by unknown authority

**Comment by scholzj** [comment]
I'm not sure what exactly this means. The last part of the error suggests that your registry is using some unsigned or invalid certificate. That would be a bit weird it you used it with some public registry such as Docker Hub, Quay or the GitHub. But assuming you are on Kubernetes, I guess you can try this option: https://github.com/GoogleContainerTools/kaniko#--insecure ... you can set it in the https://strimzi.io/docs/operators/latest/full/using.html#type-DockerOutput-reference
